src/app/hooks/useDetectOrdinalInSignPsbt.ts

```diff
@@ -15,21 +16,30 @@ const useDetectOrdinalInSignPsbt = (parsedPsbt: '' | ParsedPSBT) => {
       setLoading(true);
       await Promise.all(
         parsedPsbt.inputs.map(async (input) => {
-          const data = await getUtxoOrdinalBundle(input.txid, input.index);
-          console.log({ data });
-
-          const bundle = mapRareSatsAPIResponseToRareSats(data);
-          bundle.items.forEach((item) => {
-            // we don't show unknown items for now
-            if (item.type === 'unknown') {
-              return;
+          try {
+            const data = await getUtxoOrdinalBundle(input.txid, input.index);
+
+            const bundle = mapRareSatsAPIResponseToRareSats(data);
+            bundle.items.forEach((item) => {
+              // we don't show unknown items for now
+              if (item.type === 'unknown') {
+                return;
+              }
+              bundleItems.push(item);
+            });
+          } catch (e) {
+            // we get back a 404 if the UTXO is not found, so it is likely this is a UTXO from an unpublished txn
+            if (!isAxiosError(e) || e.status !== 404) {
+              // rethrow error if response was not 404
+              throw e;
```

Comment:
so this maps to a rejected promise?

Reply:
I guess it doesn't matter since we don't use the result of the Promise.all

Reply:
Yeah. If this happens then the function will returned a rejected promise and we'd technically go back to the current behaviour with the endless loading indicator. 

Maybe we should make a ticket to show an error stating that we are having difficulty confirming the Ordinals/rare sats in the UTXOs so it's risky to publish the txn in this scenario.